cmd/cstor-pool-mgmt/pool/pool.go

```diff
@@ -90,6 +97,29 @@ func CreatePool(cStorPool *apis.CStorPool) error {
 	return nil
 }
 
+func validateDiskCount(pool *apis.CStorPool) error {
+	poolType := pool.Spec.PoolSpec.PoolType
+	diskCount := len(pool.Spec.Disks.DiskList)
```

Comment:
is this the same DiskList variable used for auto provisioning and manual way?

Reply:
If get you question right @vishnuitta .
This is not the same. It is from the CstorPool object which is having disks (device Id) of the node to which it is associated.

Reply:
got it.. @utkarshmani1997 lets move validation things to spc watcher code, so that, auto/manual thing also gets fixed there

Reply:
@vishnuitta as discussed with @sonasingh46 i got advice to do validation here ?